detection_rules/schemas/definitions.py

```diff
@@ -34,7 +34,8 @@
 SAVED_QUERY = 'saved_query'
 QUERY = 'query'
 
-
+# we had a bad rule ID make it in before tightening up the pattern, and so we have to let it bypass
```

Comment:
Can you explain why for future reference? Also will this list just continue to grow or will we ever resolve the bad rules to shrink the list?

Reply:
this should never happen again - we had a rule slip through with an additional character in the UUID. We have since deprecated it and replaced it, but this accounts for that bad ID to pass validation (https://github.com/elastic/detection-rules/commit/ab17dfcc28d4b455c6730abfab8ff60a3166c800)